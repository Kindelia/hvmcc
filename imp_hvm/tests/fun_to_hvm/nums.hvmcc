function test_nums () {
    {test_nums} =
        let a = 0053246262234;
        let b = 0xffff;
        let c = 0.2356564;
        let d = 34689798134.7676454657254;
        {Pair (+ a b) (+ c d)}
}