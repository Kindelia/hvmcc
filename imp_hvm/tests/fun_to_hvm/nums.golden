(test_nums) = let a = 53246262234; let b = 65535; let c = 0.23565639999999988; let d = 34689798134.7677; (Pair (+ a b) (+ c d))